cowprotocol/monitoring/solver_approvals_4173928.sql

```diff
@@ -0,0 +1,30 @@
+with
+ranked as (
+    select
+        evt_tx_hash as tx_hash,
+        evt_block_time as block_time,
+        "from" as responsible_address,
+        spender,
+        contract_address as token,
+        a.value,
+        rank() over (partition by spender, contract_address order by evt_block_number desc, evt_index desc) as rk
+    from erc20_{{blockchain}}.evt_Approval as a
+    inner join {{blockchain}}.transactions on evt_tx_hash = hash
+    where owner = 0x9008D19f58AAbD9eD0D60971565AA8510560ab41
+)
+
+select --noqa: ST06
```

Comment:
what does this comment mean?

Reply:
It disables a formatting check. The check is that in all select statements, one first has simple column names, then computed columns. We disable it often since deviating ordering of columns makes some queries easier to read.